### PR TITLE
[abi][vfabia64] Specify the value of inactive lanes in output.

### PR DESCRIPTION
**Pull request by fpetrogalli**
The issue has been discussed in
https://github.com/ARM-software/software-standards/issues/2

### CODE REVIEW

File: abi/vfabia64/vfabia64.rst [hunk 1]
```diff
@@ -1489,6 +1489,12 @@ Masked signature generation for ``[not]inbranch`` clause.
 |``#pragma omp declare simd notinbranch`` |No     |Yes     |Yes   |No      |
 +-----------------------------------------+-------+--------+------+--------+
 
+Return value and masking
+------------------------
+
+The inactive lanes of return values of masked functions is ``undef``.
```
Comment (rsandifo-arm) [inline abi/vfabia64/vfabia64.rst]: `undef` sounds like LLVM terminology and isn't used elsewhere in the doc.  Maybe just "are undefined"?

Should we document this for arguments too?

Comment (fpetrogalli) [reply abi/vfabia64/vfabia64.rst]: Done.

File: abi/vfabia64/vfabia64.rst [hunk 2]
```diff
@@ -1489,6 +1489,13 @@ Masked signature generation for ``[not]inbranch`` clause.
 |``#pragma omp declare simd notinbranch`` |No     |Yes     |Yes   |No      |
 +-----------------------------------------+-------+--------+------+--------+
 
+Inactive lanes
+--------------
+
+The values of the inactive lanes of the input parameters and the
```
Comment (rsandifo-arm) [inline abi/vfabia64/vfabia64.rst]: Sorry to nit-pick, but maybe:

In a masked vector function, the contents of the inactive lanes of the input parameters and the inactive lanes of the return value are undefined.

Comment (fpetrogalli) [reply abi/vfabia64/vfabia64.rst]: Done

**Comment by fpetrogalli** [comment]
@jakubjelinek, we are in the process of adding the behavior introduced in this patch to the vector function ABI for AArch64. Do you foresee any problems in term of `libgomp` or the OpenMP standard?

**Pull request merged**